vars/runPactProviderTests.groovy

```diff
@@ -2,15 +2,14 @@
 def call( String providerProjectName,
           String consumerTag) {
 
-    gitClone(gitUriFromProjectName(providerProjectName))
-    def providerSha = sh(script: "cd ${providerProjectName} && git rev-parse HEAD", returnStdout: true).trim()
-
     withCredentials([
             string(credentialsId: 'pact_broker_username', variable: 'PACT_BROKER_USERNAME'),
             string(credentialsId: 'pact_broker_password', variable: 'PACT_BROKER_PASSWORD')]
     ) {
         sh """
-            set -ue pipefail
+            set -ueo pipefail
+            rm -rf ${providerProjectName}
+            git clone git@github.com:alphagov/${providerProjectName}.git
             cd ${providerProjectName}
             export DOCKER_HOST=unix:///var/run/docker.sock
             mvn clean test -DrunContractTests -DPACT_BROKER_USERNAME=${PACT_BROKER_USERNAME} -DPACT_BROKER_PASSWORD=${PACT_BROKER_PASSWORD} -DPACT_CONSUMER_TAG=${consumerTag} -Dpact.provider.version=${providerSha} -Dpact.verifier.publishResults=true
```

Comment:
don't we need to resolve the providerSha here?

Reply:
Yes we do.. fix coming up